frontend/src/components/buttons/ButtonEdit.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-btn class="px-3 px-sm-4" color="primary" v-bind="$attrs" v-on="$listeners">
     <v-icon :left="$vuetify.breakpoint.smAndUp" size="150%">{{ icon }}</v-icon>
-    <span class="d-none d-sm-block">
+    <span class="d-sr-only d-sm-block">
       <slot>{{ $tc('global.button.edit') }}</slot>
     </span>
```

Comment:
Why did you do that? For me this hides the text on desktop

Reply:
Hm the goal was to have it always visible for screen readers, and on devices with size small and upwards. Didn't test it though. How would you do it?

Reply:
It would need a `d-sm-not-sr-only`

Reply:
Yes, maybe aria-label is easiest here. Will do that for now.

Reply:
Because this component has a slot, it was easier to add a separate (duplicate) instance of `<slot />` which is only displayed on screen readers, and only if the original slot isn't already displayed.